client/src/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,157 @@
+/* eslint-disable react/prop-types */
+
+import React from 'react';
```

Comment:
It is recommended to import React this way 
```suggestion
import * as React from 'react';
```
It will work either way, here is a good explanation: https://stackoverflow.com/a/55287189